fuzzers/libfuzzer/builder.Dockerfile

```diff
@@ -15,12 +15,21 @@
 ARG parent_image
 FROM $parent_image
 
+# Install dependencies.
+RUN apt-get update && \
+    apt-get remove -y llvm-10 && \
+    apt-get install -y \
+        build-essential \
+        lsb-release wget software-properties-common gnupg && \
+    wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && ./llvm.sh 18
```

Comment:
this will install binaries into /usr/bin but won't make /usr/bin/clang point to /usr/bin/clang-18

so if you run clang then older one, if it exists, will still be used
to fix this you need something like this https://gist.github.com/tokatoka/26f4ba95991c6e33139999976332aa8e or use update-alternative (since this is ubuntu